### PR TITLE
Corrected view

### PR DESCRIPTION
**Pull request by aashna27**
#866 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
Ah, I think someone was trying to have the crop module show a centered image. Is that incompatible with the UI though? Or could we have the cropping UI also be centered?

**Comment by aashna27** [comment]
> 
> 
> Ah, I think someone was trying to have the crop module show a centered image. Is that incompatible with the UI though? Or could we have the cropping UI also be centered?

yeahh it wasnt happening , with the made changes!

**Comment by HarshKhandeparkar** [comment]
@aashna27 can we vertically translate the image(by 50%?) ? No need to center horizontally. Atleast vertically? Is that possible? Also what will happen if the image is resized?

**Comment by Harshithpabbati** [comment]
Ok i will check it.

**Comment by Harshithpabbati** [comment]
Yeah It's nice even with the smaller screens.
old

**Comment by HarshKhandeparkar** [comment]
Add the label

**Comment by Harshithpabbati** [comment]
How abt the crop module ???

**Comment by Harshithpabbati** [comment]
Some issue is there right for centering the image in the crop module

**Comment by Harshithpabbati** [comment]
Had a problem...

**Comment by Harshithpabbati** [comment]
@aashna27 The image is not hiding when we press the toggle.

**Comment by HarshKhandeparkar** [comment]
Yes i saw that as well. It's not a problem of this pr. Check out beta.sequencer.publiclab.org. pr #881 fixes the issue.

File: examples/lib/defaultHtmlStepUi.js [hunk 1]
```diff
@@ -39,9 +39,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
     </form>\
     <div class="col-md-8 cal step-column">\
```
[the remark this answers is not in the document]
Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlStepUi.js]: Why? Since this is @mgroovyank's first issue, let's not ask for any more changes.

**Comment by Harshithpabbati** [comment]
> Yes i saw that as well. It's not a problem of this pr. Check out beta.sequencer.publiclab.org. pr #881 fixes the issue.

If we remove that step-column from the div then it will work. There is no use for the step-column.

File: examples/lib/defaultHtmlStepUi.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlStepUi.js]: We can ask for that change in another pr or in a separate issue. Let's merge this first

[the remark this answers is not in the document]
Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlStepUi.js]: Oh wait. This is aashna's pr. Sorry. I am all confused lol😂

[the remark this answers is not in the document]
Comment (Harshithpabbati) [reply examples/lib/defaultHtmlStepUi.js]: Yeah but if we romove that line then it will work fine .

[the remark this answers is not in the document]
Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlStepUi.js]: That other pr by @mgroovyank already fixes ths issue so lets merge taht one first and think about this one later

**Comment by Harshithpabbati** [comment]
We need the changes here.

**Comment by Harshithpabbati** [comment]
@aashna27 wait for https://github.com/publiclab/image-sequencer/pull/881 this to merge and then send a update this.

**Comment by jywarren** [comment]
Yes, i think that was the issue - we can re-open the crop module one and try to better and more narrowly describe the problem - that when images change size, they display to the top-left; but that any solution needs to be sure not to break other module display. Thanks!

**Pull request merged**